common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java

```diff
@@ -183,6 +187,14 @@ public void setComplianceEmailTemplatesV2Enabled(boolean complianceEmailTemplate
         this.complianceEmailTemplatesV2Enabled = complianceEmailTemplatesV2Enabled;
     }
 
+    public boolean isDriftEmailTemplatesV2Enabled() {
+        return driftEmailTemplatesV2Enabled;
+    }
+
+    public void setDriftEmailTemplatesV2Enabled(boolean driftEmailTemplatesV2Enabled) {
+        this.driftEmailTemplatesV2Enabled = driftEmailTemplatesV2Enabled;
```

Comment:
I probably missed this in previous PRs (if any). When the feature is only meant to be switched on tests, we add a check to prevent updating it on the application.

```suggestion
        checkTestLaunchMode();
        this.driftEmailTemplatesV2Enabled = driftEmailTemplatesV2Enabled;
```